lib/order.rb

```diff
@@ -0,0 +1,83 @@
+require './lib/menu'
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Order
+
+  # include SendText
+
+  attr_reader :items, :total, :list
+
+  def initialize
+    @items = []
+    @dishes = Menu::DISHES
+    @total = 0
+  end
+
+  # Order an item with quantity
+  def order(item, quantity = 1)
+    check_item(item)
+    item_quant = [item, quantity]
+    @items << item_quant
+    calc_price
+    puts "You have added #{quantity}x #{item} to your order."
+    puts "Your subtotal is: £#{@total}"
+  end
+
+  # List current order with subtotal
+  def list
+    @items.each do |item|
+      puts "#{item[1]}x #{item[0]}"
+    end
+    puts "Your total today is £#{@total}"
+  end
+
+  # Calculate subtotal of order
+  def calc_price
+    @items.each do |item|
+      @dishes.each do |dish|
+        if item[0] == dish.name
+          @total += (dish.price * item[1])
+        end
+      end
+    end
+  end
+
```

Comment:
maybe theres a simpler way to do this? maybe items could contain dish objects with a tied quantity instance variable? to be fair I tried to do this but couldn't get it implemented that way. not sure if this is much of an issue anyway